### DISCUSSION TITLE
Google Maps interference Fermata FS mirror

### DISCUSSION THREAD

**Discussion by SOFO888** [AndreyPavlenko/Fermata#398] [Ideas]
Hey Andrey, 

Thanks again for the FS mirror app. As FS mirror is built as a navigation app, it is the first app to start when connected to Android Auto. I love this function. 

I noticed the following interference with Google Maps. As soon as I click on a Google Maps link on the phone, Google Maps on the phone starts and not on the headunit. That means the links are not sent to Google Maps on the headunit. 

If I open the app on the headunit and click on a Google Maps link afterwards, the link is sent to the Google Maps app on the headunit. 

The reason is clear. Android Auto uses the last navigation app for Maps links. 

Is it possible to integrate a solution to this topic? 
I'm thinking of starting Google Maps automatically on the headunit as soon as a Google Maps link is clicked. Then the last navigation app is Google Maps and the links will be opened on the headunit on Google Maps. 

Thank you.

**Comment by AndreyPavlenko** [comment]
I doubt I have control over opening links by gmap. As a workaround, you can use Fermata Mirror instead of Fermata FS Mirror.

**Reply by SOFO888** [reply]
Ok, thanks for the fast reply.

**Comment by SOFO888** [comment]
Hey Andrey, 
I googled and found the following information: 

https://stackoverflow.com/questions/67497050/android-auto-how-to-explicitly-open-google-map-for-navigation-to-show-on-car-s

https://medium.com/@loicteyssier/bring-your-android-app-into-your-car-part-2-cafe-finder-android-auto-app-7a8063b098e2

Do you think that this could help? 

Thank you.

**Reply by AndreyPavlenko** [reply]
But there is no answer. As I remember, I tried to implement gmap links opening, but it worked only if gmap is started.
Why you can't just use the phone version of gmap over the mirror? It seems very similar to the mobile version and works quite good.

**Reply by SOFO888** [reply]
Thank you for the anwer. 

Let me explain. 

The reason why I want to use gmaps through the headunit is that navigation will not start on the headunit when I click on a gmaps link. Fermata stays open and there is no navigation at all. 

Background is that I made a Tasker script. When I drive, my codriver can send me the destination by Whatsapp. The Tasker script clicks on the gmaps link. Without Fermata gmaps will open on the headunit and navigation starts. 
With Fermata nothing happens as the link is sent to Fermata and not interpreted.
